tifeatures/templates/collection.html

```diff
@@ -1,5 +1,18 @@
 {% include "header.html" %}
 
+<nav aria-label="breadcrumb">
+  <ol class="breadcrumb bg-light">
+    {% for crumb in crumbs %}
+      {% if not loop.last %}
+    <li class="breadcrumb-item"><a href="{{ crumb.url }}/">{{ crumb.part }}</a></li>
+      {% else %}<li class="breadcrumb-item active" aria-current="page">{{ crumb.part }}</li>
+      {% endif %}
+    {% endfor %}
+
+    <li class="ml-auto json-link"><a target="_blank" href="{{ url }}?f=json">JSON</a></li>
+  </ol>
+</nav>
```

Comment:
I had to move this outside the header.html because for `items` and `item` we want `GeoJSON` not `JSON`